starter-code/pizza.js

```diff
@@ -1 +1,123 @@
 // Write your Pizza Builder JavaScript in this file.
+// $( document ).ready(function() {
+//   $('.btn').click(function(){
+//     $(this).addClass('on');
+//   });
+// });
+
+// // hide/unhide/toggle
+
+// $
+// .add()
+// .append()
+// .remove
+
+// :button
+// .click()
+
+
+// var initialPrice = 10;
+// var totalPrice =
+
+
+$(".btn-pepperonni").click(function(){
+  $(".pep").toggle();
+  $(".price ul li:nth-child(1)").toggle();
```

Comment:
Here I see you select the list item by its position in the list. It works now, but if there were any changes to happen to the list, like new ingredients or simply a reorganization, this would malfunction.
You could select the <li> depending on the text it contains, as you did for the sauce and crust default settings:
```javascript
$(".price li:contains('sauce')").hide();
$(".price li:contains('crust')").hide();
```

This way the code is much more robust :)